src/Orleans/Streams/PersistentStreams/StreamQueueBalancerType.cs

```diff
@@ -1,19 +1,22 @@
 namespace Orleans.Streams
 {
-    public enum StreamQueueBalancerType
+    /// <summary>
+    /// Built-in stream queue balancer type which is supported natively in orleans
+    /// </summary>
+    public static class BuiltInStreamQueueBalancerType
```

Comment:
let's leave the enum, and have this class expose string for the enums.  That way legacy code/tests can still use these enum values.

In the config object we can add a 'string QueueBalancer' property and change the BalancerType to be a passthrough that setts the  QueueBalancer'.  The property in the property bag would remain "BalancerType"